src/pages/index.js

```diff
@@ -0,0 +1,331 @@
+
+//Edit modal
+const editModal = document.querySelector("#edit-modal");
+const editModalCloseButton = editModal.querySelector(".modal__close-btn");
+const editModalNameInput = editModal.querySelector("#profile-name-input");
+const editModalDescriptionInput = editModal.querySelector(
+  "#profile-description-input"
+);
+const editFormInput = document.querySelector(".modal__form");
+
+const cardModal = document.querySelector("#add-card-modal");
+const cardModalCloseBttn = cardModal.querySelector(".modal__close-btn");
+const cardLink = document.querySelector(".card__image");
+const cardName = document.querySelector(".card__title");
+const cardModalLinkInput = cardModal.querySelector("#add-card-link-input");
+const cardModalNameInput = cardModal.querySelector("#add-card-name-input");
+const cardFormInput = cardModal.querySelector(".modal__form");
+const cardSubmitBtn = cardModal.querySelector(".modal__submit-btn");
+
+//Card elements
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+
+let selectedCard, selectedCardId;
+let currentUserId;
+
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageEl = previewModal.querySelector(".modal__image");
+const previewModalCaptionEl = previewModal.querySelector(".modal__caption");
+const previewModalCloseButton = previewModal.querySelector(".modal__close-btn");
+const deleteCancelButton = document.getElementById("cancel-delete-btn");
+
+function getCardElement(data) {
+  const cardElement = cardTemplate.content
+    .querySelector(".card")
+    .cloneNode(true);
+
+  const cardNamedl = cardElement.querySelector(".card__title");
+  const cardImage = cardElement.querySelector(".card__image");
+  const likeButton = cardElement.querySelector(".card__like-btn");
+  const deleteButton = cardElement.querySelector(".card__delete-btn");
+
+  cardNamedl.textContent = data.name;
+  console.log("Card data:", data);
+  console.log("Card image element src:", cardImage.src);
+
+  console.log("Card name:", data.name);
+  console.log("Card link:", data.link);
+
+  cardImage.src = data.link;
+  cardImage.alt = data.name;
+
+  const isLikedByCurrentUser = data.isLiked === true;
+
+  if (isLikedByCurrentUser) {
+    likeButton.classList.add("card__like-btn_liked");
+  }
+
+  if (
+    Array.isArray(data.likes) &&
+    data.likes.some((user) => user._id === currentUserId)
+  ) {
+    likeButton.classList.add("card__like-btn_liked");
+  }
+
+  likeButton.addEventListener("click", (evt) => {
+    handleLike(evt, data._id);
+  });
+
+  deleteButton.addEventListener("click", () =>
+    handleDeleteCard(cardElement, data._id)
+  );
+
+  cardImage.addEventListener("click", () => {
+    openModal(previewModal);
+    previewModalCaptionEl.textContent = data.name;
+    previewModalImageEl.src = data.link;
+    previewModalImageEl.alt = data.name;
+  });
+
+  return cardElement;
+}
+
+function handleeditFormSubmit(evt) {
```

Comment:
`handleeditFormSubmit`

Every variable in the project should be in `camelCase`